portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
   </head>
-  <body>
+  <body onload="createMap();">
```

Comment:
avoid missing js with html. prefer document.addEventListener('DOMContentLoaded', fn) or window.addEventListener('load', fn) over this

Reply:
I have done the changes, please review it again